module/input/NatNet/src/NatNet.cpp

```diff
@@ -372,32 +383,32 @@ namespace module::input {
         std::string name(&packet.data);
-        const char* appVersion    = &packet.data + 256;
-        const char* natNetVersion = appVersion + 4;
+        const char* app_version     = &packet.data + 256;
+        const char* nat_net_version = app_version + 4;
 
         // Update our version number
-        version =
-            (natNetVersion[0] << 24) | (natNetVersion[1] << 16) | (natNetVersion[2] << 8) | (natNetVersion[3] << 0);
+        version = (nat_net_version[0] << 24) | (nat_net_version[1] << 16) | (nat_net_version[2] << 8)
+                  | (nat_net_version[3] << 0);
 
         // Make our app version a string (removing trailing 0 version numbers)
-        std::string strAppVersion = std::to_string(int(appVersion[0]))
-                                    + (appVersion[1] == 0 ? "" : "." + std::to_string(appVersion[1]))
-                                    + (appVersion[2] == 0 ? "" : "." + std::to_string(appVersion[2]))
-                                    + (appVersion[3] == 0 ? "" : "." + std::to_string(appVersion[3]));
+        std::string str_app_version = std::to_string(int(app_version[0]))
+                                      + (app_version[1] == 0 ? "" : "." + std::to_string(app_version[1]))
+                                      + (app_version[2] == 0 ? "" : "." + std::to_string(app_version[2]))
+                                      + (app_version[3] == 0 ? "" : "." + std::to_string(app_version[3]));
 
-        // Make our natNetVersion a string
-        std::string strNatVersion = std::to_string(natNetVersion[0])
-                                    + (natNetVersion[1] == 0 ? "" : "." + std::to_string(natNetVersion[1]))
-                                    + (natNetVersion[2] == 0 ? "" : "." + std::to_string(natNetVersion[2]))
-                                    + (natNetVersion[3] == 0 ? "" : "." + std::to_string(natNetVersion[3]));
+        // Make our nat_net_version a string
+        std::string str_nat_version = std::to_string(nat_net_version[0])
+                                      + (nat_net_version[1] == 0 ? "" : "." + std::to_string(nat_net_version[1]))
+                                      + (nat_net_version[2] == 0 ? "" : "." + std::to_string(nat_net_version[2]))
+                                      + (nat_net_version[3] == 0 ? "" : "." + std::to_string(nat_net_version[3]));
 
         // Make our remote into an IP
-        std::string strRemote = std::to_string((remote >> 24) & 0xFF) + "." + std::to_string((remote >> 16) & 0xFF)
-                                + "." + std::to_string((remote >> 8) & 0xFF) + "."
-                                + std::to_string((remote >> 0) & 0xFF);
+        std::string str_remote = std::to_string((remote >> 24) & 0xFF) + "." + std::to_string((remote >> 16) & 0xFF)
+                                 + "." + std::to_string((remote >> 8) & 0xFF) + "."
+                                 + std::to_string((remote >> 0) & 0xFF);
```

Comment:
This is a job for fmt::format for sure

Reply:
Also, I think something like this will work for the IP address?

```cpp  
  std::array<char, INET_ADDRSTRLEN> str{};
  inet_ntop(AF_INET, &remote, str.data(), str.size());
  std::string IP = str.data();
```